src/com/epam/javatrainings/classdecomposition/pizzeria_palmetto/services/PlacingAnOrderService.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services;
+
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Ingredients;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Pizza;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.PlacingAnOrder;
+
+import static com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services.Constants.ASTERISK;
+import static com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services.Constants.LINE;
+
+public class PlacingAnOrderService implements OrderCreator, Validator {
+  private final PlacingAnOrder order;
+
+  public PlacingAnOrderService(PlacingAnOrder order) {
+    this.order = order;
+  }
+
+  public void printCheck() {
+    System.out.println(ASTERISK);
+    System.out.println("Order: " + order.getOrderNumber());
+    System.out.println("Client: " + order.getCustomer().getNumberCustomer());
+    System.out.println("Time: " + order.getLocalTime());
+    for (Pizza pizza : order.getOrderItems()) {
+      System.out.println("Name: " + pizza.getName());
+      System.out.println(LINE);
+      System.out.println(
+          "Pizza Base " + "(" + pizza.getType() + ")" + pizza.getType().getCost() + " €");
+
+      for (String ingredient : Pizza.ingredientsList) {
+        for (Ingredients ingredients : Ingredients.values()) {
+          if (ingredients.getKey().equals(ingredient)) {
+            System.out.println(ingredient + " " + ingredients.getCost());
+          }
+        }
+      }
```

Comment:
can be more simplify:
pizza.ingredientsList.forEach(ingredients -> System.out.println(ingredients.getKey() + " " + ingredients.getCost()));